### DISCUSSION TITLE
Quick question about learning rates...

### DISCUSSION THREAD

**Discussion by rafaelblevin821** [BindsNET/bindsnet#614] [Q&A] [answered]
So nu accepts a pair of learning rates for both the pre-synaptic and post-synaptic weight updates in the PostPre learning rule. If only one learning rate is defined, does the network use this value for both pre-synaptic and post-synaptic weight calculations, or does it only use this only for the pre-synaptic weight calculation and no learning rate multiplier for the post-synaptic weight calculation?

**Comment by Hananel-Hazan** [comment, marked as answer]
From the code [documentation](https://github.com/BindsNET/bindsnet/blob/989ebaf2a3bff4d5a3a8f6c509ffafeaf9a29fad/bindsnet/learning/learning.py#L167)
```
:param nu: Single or pair of learning rates for pre- and post-synaptic events. It also accepts a pair of tensors to individualize learning rates of each neuron. In this case, their shape should be the same size as the connection weights.
```

Looking on the code [here](https://github.com/BindsNET/bindsnet/blob/989ebaf2a3bff4d5a3a8f6c509ffafeaf9a29fad/bindsnet/learning/learning.py#L62) if a nu is set with single int or float, then both pre and post has the same value.

**Reply by rafaelblevin821** [reply]
@Hananel-Hazan Thank you
